src/repository/SensorEventRepository.ts

```diff
@@ -22,6 +23,7 @@ export class SensorEventRepository {
                   $gte: from_date,
                   $lt: from_date === to_date ? to_date! + 1 : to_date,
                 },
+            data:(!is_device)?(undefined as any):{device_type:{ $ne: 'Dashboard'}}    
```

Comment:
Instead of this, call the function normally, and use `res.filter(x => x.data.device_type != "Dashboard")` as this method requires CouchDB to have an index including this parameter otherwise will slow down queries.

Reply:
This is done by giving a limit and filtering for dashboard